src/gatsby/onPostBuild.ts

```diff
@@ -67,6 +67,37 @@ export default async ({ graphql }) => {
   await writeJson(output, nodes, platformRegistry);
 };
 
+const parsePathSlug = (slug: string) => {
+  if (slug.includes("performance-onboarding")) {
+    const pathMatch = slug.match(
+      /^\/(?<platform>[^/]+)\/performance-onboarding\/(?<sub_platform>[^/]+)\/(?<step>[^/]+)\/$/
+    );
```

Comment:
should we throw an error if we don't match here?

Reply:
@AbhiPrasad good catch. I've thrown an error here.